bin/run.sh

```diff
@@ -7,21 +7,54 @@ cd "$( dirname "${BASH_SOURCE[0]}" )/.."
 
 echo() { builtin echo "[SCITRAN] $@"; }
 
-
+usage() {
+  echo "Usage: $0 [-c <config file>] [-d|-s]"
+  echo "   -d Daemonize"
+  echo "   -s Stop running daemon"
+  exit 1
+}
+
+waitfor() {
+  while ps -p $@; do
+    sleep 0.5
+  done
+}
+
+#Default is to start dev server unless start/stop is specified
+DEV=1
+# A colon suffix ":" means that option takes an argument
+# See getopt man page for more info
+while getopts "c:ds" opt; do
+  case "$opt" in
+    c)
+      echo "Using config file $OPTARG"
+      EXISTING_ENV=$(env | grep "SCITRAN_" | cat)
+      source "$OPTARG"
+      eval "$EXISTING_ENV"
+      ;;
+    d)
+      echo "Starting scitran in daemon mode..."
+      START=1
+      DEV=0
+      ;;
+    s)
+      echo "Stopping scitran..."
+      STOP=1
+      DEV=0
+      ;;
+    \?|:)
+      echo "Error parsing arguments!";
+      usage
+      exit 1
+      ;;
+
+  esac
+done
+
+
+# Exported default config values
 set -o allexport
-
-if [ "$#" -eq 1 ]; then
-    EXISTING_ENV=$(env | grep "SCITRAN_" | cat)
-    source "$1"
-    eval "$EXISTING_ENV"
-fi
-if [ "$#" -gt 1 ]; then
-    echo "Usage: $0 [config file]"
-    exit 1
-fi
-
-
-# Default config values
+SCITRAN_CORE_INSECURE=${SCITRAN_CORE_INSECURE:-"true"}
```

Comment:
Yep, we could do that. I thought this would be easier for newbies - the tests fail without this setting.